### PR TITLE
Models (drink menu employee)

### PR DESCRIPTION
**Pull request by domisobi**

### CODE REVIEW

File: server/db/models/Drink.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      }, 
+      type: {
+        type: DataTypes.STRING
+      },
+      price: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        validate: {
+          notEmpty: true,
+        }
```
Comment (justinmattos) [inline server/db/models/Drink.js]: I think this needs `type: FLOAT` to store decimal values?

File: server/db/models/Menu.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    type: DataTypes.UUID,
+    defaultValue: DataTypes.UUIDV4,
+    primaryKey: true,
+    },
+name: {
+    type: DataTypes.STRING, // is menu name just venue name?
+    allowNull: false,
+    validate: { 
+        notEmpty: true,
+    },
```
Comment (justinmattos) [inline server/db/models/Menu.js]: I think we talked about maybe the menu could be a way for restaurants to categorize their offerings, so one venue could have a Wine menu and a Beer menu and a Cocktail menu that users could browse individually

**Review by justinmattos — APPROVED** [review]
Some small comments, but good to merge I think

File: server/db/models/Drink.js (hunk 1 above)
Comment (domisobi) [reply server/db/models/Drink.js]: yes, thank you

File: server/db/models/Menu.js (hunk 2 above)
Comment (domisobi) [reply server/db/models/Menu.js]: yes we did! I'll fix the other thing and remove the comment here.

**Pull request merged**